src/NuGetPackageVerifier/CompositeRules/NoTimestampPackagesCompositeRule.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class NoTimestampPackagesCompositeRule : IPackageVerifierRule
+    {
+        private IPackageVerifierRule[] _rules = new IPackageVerifierRule[]
+        {
+            new PrereleaseDependenciesVersionRule()
```

Comment:
@Eilon See [this](https://github.com/aspnet/BuildTools/pull/144#discussion_r95694636). Currently `SigningVerificationCompositeRule` is the only one in the `NugetPackageVerifier.json` file.